### PR TITLE
[Miniconda] - Auto install vuln packages

### PR DESCRIPTION
**Pull request by gauravsaini04**
**Dev container name**:
 
 * Miniconda
 
 **Description**:

This PR adds the following functionality : 

* Auto install vulnerable packages from conda channel / pip pkg managers

### CODE REVIEW

File: src/miniconda/.devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,15 @@
 FROM continuumio/miniconda3 as upstream
 
-# Temporary: Upgrade python packages due to mentioned CVEs
-# They are installed by the base image (continuumio/miniconda3) which does not have the patch.
-RUN conda install \ 
-    # https://github.com/advisories/GHSA-jjg7-2v4v-x38h
-    idna==3.7
+# Temporary: Upgrade python packages
```
[the remark this answers is not in the document]
Comment (gauravsaini04) [reply src/miniconda/.devcontainer/Dockerfile:3]: sure have changed #1079 according to your review comments

**Comment by samruddhikhandale** [comment]
As https://github.com/devcontainers/images/pull/1079 is merged, let's update this PR to reflect the changes.

**Comment by gauravsaini04** [comment]
> As #1079 is merged, let's update this PR to reflect the changes.

done

File: src/miniconda/.devcontainer/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,9 @@
...
-RUN conda install \ 
-    # https://github.com/advisories/GHSA-jjg7-2v4v-x38h
-    idna==3.7
-
-# Temporary: Upgrade python packages using pip package manager
-# RUN python3 -m pip install --upgrade \
-    # https://github.com/advisories/
-    # package==version
+# Temporary: Upgrade python packages
+# COPY ./apply_security_patches.sh /tmp/apply_security_patches.sh
```
Comment (samruddhikhandale) [inline src/miniconda/.devcontainer/Dockerfile:4]: Why is this commented out?
Do we not need `idna==3.7` patch anymore?

Comment (gauravsaini04) [reply src/miniconda/.devcontainer/Dockerfile:4]: yeah its shipping from upstream now

**Review by samruddhikhandale — APPROVED** [review]

**Pull request merged**